lcm/service/lcm/constants.go

```diff
@@ -78,8 +78,8 @@ const (
 	loadModelMemInMB=50
 	loadTrainingDataMilliCPU=20
 	loadTrainingDataMemInMB=300
-	logCollectorMilliCPU=20
-	logCollectorMemInMB=100
+	logCollectorMilliCPU=60
+	logCollectorMemInMB=300
 	controllerMilliCPU=20
```

Comment:
How did we come up with these numbers?

Reply:
Similar to the Load Training Data memory, giving log collector 3x capacity seems to be the right fix. 2x capacity will work on our example but it will be insufficient again with a bigger job. I am working on moving these constants to the helm chart, so in the future users can just customize these number based on their needs.